testing/testcafe/tests/editors/autocomplete/common.ts

```diff
@@ -0,0 +1,31 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import url from '../../../helpers/getPageUrl';
+import Autocomplete from '../../../model/autocomplete';
+import createWidget from '../../../helpers/createWidget';
+import { restoreBrowserSize } from '../../../helpers/restoreBrowserSize';
+
+fixture`Autocomplete_placeholder`
+  .page(url(__dirname, '../../container.html'))
+  .afterEach(async (t) => {
+    await restoreBrowserSize(t);
```

Comment:
Do we really need a functional test for this scenario? Seems like unit test can be written easy as well

Reply:
Yes, there are no testcafe tests for placeholder.